community/duo_unix/APKBUILD

```diff
@@ -1,39 +1,38 @@
-# Contributor: Alan Lacerda <alacerda@alpinelinux.org>
-# Maintainer: Paul Morgan <jumanjiman@gmail.com>
+# Contributor: Kris Reese <ktreese@gmail.com>
+# Maintainer: Kris Reese <ktreese@gmail.com>
 pkgname=duo_unix
-pkgver=1.11.0
-pkgrel=1
-pkgdesc="duosecurity.com two-factor authentication"
-url="https://duo.com/support/documentation/duounix"
+pkgver=1.11.1
+pkgrel=0
+pkgdesc="Duo two-factor authentication for Unix systems with Pluggable Authentication Modules (PAM)"
+url="https://duo.com/docs/duounix"
 arch="all"
-license="GPL-2.0-only"
-depends=""
-depends_dev="openssl-dev zlib-dev"
-makedepends="$depends_dev automake autoconf libtool"
+license="GPL-2.0"
+depends="openssh openssh-server-pam"
+makedepends="openssl-dev linux-pam-dev openssh-server-pam libtool"
+install=""
+subpackages="$pkgname-doc"
+source="https://dl.duosecurity.com/$pkgname-$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
 options="suid"
-subpackages="$pkgname-dev $pkgname-doc"
-source="https://github.com/duosecurity/duo_unix/archive/$pkgname-$pkgver.tar.gz"
-builddir="$srcdir"/$pkgname-$pkgname-$pkgver
 
 prepare() {
 	default_prepare
-	./bootstrap
 }
+
 build() {
 	cd "$builddir"
-	./configure \
-		--build=$CBUILD \
-		--host=$CHOST \
-		--prefix=/usr \
-		--sysconfdir=/etc/duo \
-		--mandir=/usr/share/man \
-		--localstatedir=/var
+	./configure --with-pam --prefix=/usr
 	make
 }
 
 package() {
 	cd "$builddir"
+	mkdir -p "$pkgdir"
 	make DESTDIR="$pkgdir" install
 }
 
-sha512sums="9a58de49208511a12942575959d0c3a438a216f25b478035cbf2bbd04d2c644ed45f32d77ef947d8b181e7005fd5954990aab3c9d9301fa3f790918faad43e6a  duo_unix-1.11.0.tar.gz"
+check() {
+	cd "$builddir"
+}
+
```

Comment:
This function does effectively nothing. If a test suite does not exist add

"options=!check" # no test suite

among the APKBUILD variables at the beginning. Otherwise please run the tests here.

Reply:
Resolved via be68e705ba